assets/css/main.scss

```diff
@@ -0,0 +1,234 @@
+---
+# this ensures Jekyll reads the file to be transformed into CSS later
+# only Main files contain this front matter, not partials.
+---
+
+// Configuration
+@import "functions";
+@import "variables";
+@import "maps";
+@import "mixins";
+@import "utilities";
+
+// Layout & components
+@import "root";
+@import "reboot";
+@import "type";
+@import "images";
+@import "containers";
+@import "grid";
+@import "tables";
+@import "forms";
+@import "buttons";
+@import "transitions";
+@import "dropdown";
+@import "button-group";
+@import "nav";
+@import "navbar";
+@import "card";
+@import "accordion";
+@import "breadcrumb";
+@import "pagination";
+@import "badge";
+@import "alert";
+@import "progress";
+@import "list-group";
+@import "close";
+@import "toasts";
+@import "modal";
+@import "tooltip";
+@import "popover";
+@import "carousel";
+@import "spinners";
+@import "offcanvas";
+@import "placeholders";
+
+// Helpers
+@import "helpers";
+
+// Utilities
+@import "utilities/api";
+
+:root {
+  --bs-dark: #1b1642;
+  --bs-dark-rgb: rgb(27, 22, 66, 100);
+  --bs-body-color: #1b1642;
+  --bs-body-bg-rgb: rgb(255, 255, 255, 100);
+  --bs-body-bg: #fff;
+  --bs-body-color-rgb: 33, 37, 41;
+  --bs-link-color: #1b1642;
+  --bs-link-hover-color: black;
+  --bs-primary: #4065A9;
+  --bs-primary-rgb: rgb(64, 101, 169, 100);
+  --bs-btn-padding-x: 3rem;
+  --bs-btn-border-width: 0px;
+  --bs-btn-font-weight: 600;
+  --bs-btn-font-family: var(--bs-font-sans-serif);
+  --bs-btn-border-radius: 1000px;
+  --bs-btn-focus-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125), 0 0 0 0.25rem rgba(94, 114, 183, 0.5);
+  --bs-btn-color: white;
+  --bs-btn-line-height: 1.8rem;
+  --bs-light-rgb: 220, 230, 239;
+  --bs-light: #DCE6EF;
+}
+
+:root {
+
+  --bs-secondary: #6c757d;
+  --bs-success: #198754;
+  --bs-info: #0dcaf0;
+  --bs-warning: #ffc107;
+  --bs-danger: #dc3545;
+  --bs-font-sans-serif: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  --bs-font-monospace: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+  --bs-gradient: linear-gradient(to top left, hsl(226deg, 45%, 46%) 0%, hsl(254deg, 46%, 19%) 100%);
+  --bs-body-font-family: var(--bs-font-sans-serif);
+  --bs-body-font-size: 1rem;
+  --bs-body-font-weight: 400;
+  --bs-body-line-height: 1.5;
+  --bs-border-width: 1px;
+  --bs-border-style: solid;
+  --bs-border-color: #dee2e6;
+  --bs-border-color-translucent: rgba(0, 0, 0, 0.175);
+  --bs-border-radius: 0.375rem;
+  --bs-border-radius-sm: 0.25rem;
+  --bs-border-radius-lg: 0.5rem;
+  --bs-border-radius-xl: 1rem;
+  --bs-border-radius-2xl: 2rem;
+  --bs-border-radius-pill: 50rem;
+  --bs-heading-color: red;
+  --bs-code-color: #d63384;
+  --bs-highlight-bg: #fff3cd
+}
+
+:root {
```

Comment:
Waarom losse `:root`s?

Reply:
Dat zal ik aanpassen. Is het gevolg van overnemen (maar dus nog niet samenvoegen) van huidige app en van bestaande dipcorp.